app/models/user.rb

```diff
@@ -17,19 +17,20 @@ class User < ApplicationRecord
   devise :database_authenticatable, :invitable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
-  before_create :add_default_roles
+  after_create :add_default_group_membership!
+
+  # set default scope to exclude guest users
+  def self.default_scope
+    where(guest: false)
+  end
 
   scope :for_repository, -> {
     joins(:roles)
   }
 
+  # TODO: Need to make sure this includes users in the registered group (see #add_default_group_membership!)
```

Comment:
is this still TODO?

Reply:
No, I just verified that it's working as expected. Comment removed here: https://github.com/samvera/hyku/pull/1859/commits/7fda6ecd132d6f61906a2024c168f502f9e22be6